### PR TITLE
Add new resources to DevOps section

### PR DESCRIPTION
**Pull request by mhdamaan79**
# 🛠️ Fixes issue  (#134)
# 📇 Changes proposed 
- Added new resources inside the DevOps (Cloud & Github Repositories section) of the Markdown file.
# ✅ Check List (Mark all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] This PR contains changes in the documentation
- [x] This PR contains changes in the Code of the project
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: pages/devops-resources.md [hunk 1]
```diff
@@ -110,4 +111,5 @@
...
 | :------------: | :------------: | :------------: |
-|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
\ No newline at end of file
+|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
+|  2. DevOps Learning Hub: Tools and Tech  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/Pradumnasaraf/DevOps "DevOps Learning Hub: Tools and Tech") </center> |   [Here you go](https://github.com/Pradumnasaraf/DevOps)
```
Comment (hasnainmakada-99) [inline pages/devops-resources.md]: The Preview cover for the second resource here is same as the first one, please change it @mhdamaan79

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]
Please do the changes as follows @mhdamaan79

File: pages/devops-resources.md (hunk 1 above)
Comment (mhdamaan79) [reply pages/devops-resources.md]: The preview cover for the second resource has been modified as requested. Please review the changes. @hasnainmakada-99

Comment (hasnainmakada-99) [reply pages/devops-resources.md]: Sure @mhdamaan79

File: pages/devops-resources.md [hunk 2]
```diff
@@ -110,4 +111,5 @@
...
 |Description | Preview   | Link to get started with it   |
 | :------------: | :------------: | :------------: |
-|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
\ No newline at end of file
+|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
+|  2. DevOps Learning Hub: Tools and Tech  | <center> [![open source](https://ik.imagekit.io/hfclyehqd/OS_Bag/DevOps_-_Pradumnasaraf.png?updatedAt=1684307301720 "DevOps")](https://github.com/Pradumnasaraf/DevOps "DevOps Learning Hub: Tools and Tech") </center> |   [Here you go](https://github.com/Pradumnasaraf/DevOps)
```
Comment (hasnainmakada-99) [inline pages/devops-resources.md]: Keep the cover size 320 X 180 @mhdamaan79

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]
@mhdamaan79

File: pages/devops-resources.md (hunk 2 above)
Comment (mhdamaan79) [reply pages/devops-resources.md]: I have updated the preview cover with a 320x180 image ratio. Please review the changes and let me know if any further adjustments are needed.

Comment (hasnainmakada-99) [reply pages/devops-resources.md]: Sure @mhdamaan79

File: pages/devops-resources.md [hunk 3]
```diff
@@ -110,4 +111,5 @@
...
 |Description | Preview   | Link to get started with it   |
 | :------------: | :------------: | :------------: |
-|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
\ No newline at end of file
+|  1. 90 Days of DevOps - Roadmap  | <center> [![open source](https://ik.imagekit.io/1cw2zpbjy/OSWH/Getting_started_with_DevOps_-_Banner.png?ik-sdk-version=javascript-1.4.3&updatedAt=1667725617154 "DevOps")](https://github.com/MichaelCade/90DaysOfDevOps "90 Days of DevOps - Roadmap") </center> |   [Here you go](https://github.com/MichaelCade/90DaysOfDevOps)
+|  2. DevOps Learning Hub: Tools and Tech  | <center> [![open source](https://ik.imagekit.io/hfclyehqd/OS_Bag/DevOps_-_Pradumnasaraf.jpeg?updatedAt=1684313490774 "DevOps")](https://github.com/Pradumnasaraf/DevOps "DevOps Learning Hub: Tools and Tech") </center> |   [Here you go](https://github.com/Pradumnasaraf/DevOps)
```
Comment (hasnainmakada-99) [inline pages/devops-resources.md]: Edit the description and include pradumna sarafs name in it @mhdamaan79

**Review by hasnainmakada-99 — CHANGES_REQUESTED** [review]
@mhdamaan79

File: pages/devops-resources.md (hunk 3 above)
Comment (mhdamaan79) [reply pages/devops-resources.md]: **Description:**
DevOps Learning Hub: Tools and Tech - Pradumna Saraf

Just like this? @hasnainmakada-99

Comment (hasnainmakada-99) [reply pages/devops-resources.md]: YEs @mhdamaan79

Comment (mhdamaan79) [reply pages/devops-resources.md]: Updated description with Pradumna Saraf's name included. Please review the changes @hasnainmakada-99

**Review by hasnainmakada-99 — APPROVED** [review]
Looks good to me @mhdamaan79, Thanks for your contribution

**Comment by hasnainmakada-99** [comment]
@all-contributors please add @mhdamaan79 as a contributor of tutorial

**Pull request merged**